tests/test_vxlan_vtep_vni.rb

```diff
@@ -192,6 +203,7 @@ def test_peer_list
 
   def test_suppress_arp
     vni = VxlanVtepVni.new('nve1', '6000')
+    VxlanVtep.new('nve1').host_reachability = 'evpn'
```

Comment:
I suspect that this will cause failures on certain platforms that do not support `VxlanVtep`.  Can you post the test results of N3k, N5k, N6k, N7k (F2 card), and N9k?